docs/docs/guides/web3_providers_guide/events_listening.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 0
+sidebar_label: 'Providers Events Listening'
+---
+
+# Providers Events Listening
+
+Some providers are, by design, always connected. And so they can communicate changes with the user through events. Actually, among the 3 providers, `HttpProvider` is the only one that does not support event. And the rest 2:
```

Comment:
```suggestion
Some providers are, by design, always connected and so they can communicate changes with the user through events. Actually among the 3 providers, `HttpProvider` is the only one that does not support events and the other 2:
```

Reply:
Actually, a sentence can start with 'And' :smile: 
https://www.quora.com/Is-it-grammatically-appropriate-to-put-and-after-a-period#:~:text=Yes%2C%20it%20is.,occur%20in%20sentence%2Dinitial%20position.

Reply:
It is correct, i think in terms of reading the sentence it flows better without ending the sentence but to continue but this is a nitpick from me 😄